scripts/mirror.js

```diff
@@ -220,70 +218,62 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
 
 /**
  * @param {SupportStatement} originalData
- * @param {SupportStatement} sourceData
- * @param {string} source
+ * @param {SupportStatement} chromeData
+ * @param {SupportStatement} ieData
  * @returns {SupportStatement}
  */
-const bumpEdge = (originalData, sourceData, source) => {
-  let newData = {};
+const bumpEdge = (originalData, chromeData, ieData) => {
+  if (Array.isArray(originalData)) {
+    return originalData.map(d => bumpEdge(d, chromeData, ieData));
+  }
 
-  if (source == 'ie') {
-    if (sourceData.version_removed && sourceData.version_removed !== null) {
-      newData.version_added = false;
-    } else if (sourceData.version_added !== null) {
-      newData.version_added = sourceData.version_added ? '12' : null;
-    }
+  let newData = copyStatement(originalData);
 
-    if (sourceData.notes) {
-      newData.notes = updateNotes(
-        sourceData.notes,
-        /Internet Explorer/g,
-        'Edge',
-      );
+  if (ieData) {
+    if (ieData.version_removed || ieData.version_added === false) {
+      newData.version_added = false;
+    } else if (ieData.version_added) {
+      newData.version_added = ieData.version_added ? '12' : null;
     }
-  } else if (source == 'chrome') {
-    newData = originalData == undefined ? sourceData : originalData;
-
-    let chromeFalse =
-      sourceData.version_added === false ||
-      sourceData.version_removed !== undefined;
-    let chromeNull = sourceData.version_added === null;
+  }
 
-    if (originalData === undefined) {
-      newData.version_added = chromeFalse ? false : chromeNull ? null : '≤79';
-    } else {
-      if (!chromeFalse && !chromeNull) {
-        if (originalData.version_added == true) {
-          newData.version_added = '≤18';
-        } else {
-          if (
-            sourceData.version_added == true ||
-            Number(sourceData.version_added) <= 79
-          ) {
-            if (originalData.version_added == false) {
-              newData.version_added = '79';
-            } else if (originalData.version_added == null) {
-              newData.version_added = '≤79';
-            }
-          } else {
-            newData.version_added == sourceData.version_added;
-          }
-        }
-      } else if (chromeFalse) {
-        if (originalData.version_added && !originalData.version_removed) {
-          newData.version_removed = '79';
-        }
+  let chromeFalse =
+    chromeData.version_removed || chromeData.version_added === false;
+  let chromeNull = chromeData.version_added === null;
+
+  if (!chromeFalse && !chromeNull) {
+    if (originalData.version_added === true) {
+      newData.version_added = '≤18';
+    } else if (chromeData.version_added === true) {
+      newData.version_added = true;
+    } else if (Number(chromeData.version_added) <= 79) {
+      if (
+        originalData.version_added === false ||
+        newData.version_added === false
```

Comment:
OK, so `newData.version_added === false` can happen here if it was set to false because it wasn't in IE. This makes sense, but took some pondering to understand :)

A comment for each case would be nice, but I won't insist on it.